### PR TITLE
chore: release pipeline for a new version and an alpha version

### PR DESCRIPTION
**Pull request by mstasgravitee**
AM-1064

## :computer: Add screenshots for UI
Alpha workflow
image

Release a new version
image

## :man: :woman:@mentions of the person or team responsible for reviewing proposed changes

This is a nice example: https://github.com/gravitee-io/gravitee-access-management/pull/1822

### CODE REVIEW

File: release/README.adoc [hunk 1]
```diff
@@ -124,3 +124,67 @@ flowchart LR
...
+
+== 🏁 Run release alpha version pipeline
+
+Lunch only required steps in one CircleCI pipeline to release an alpha version. There are two methods available:
+
+* npm run
+[source, shell]
+----
+npm run am-release -- \
+  --branch master --gio-version 4.3.0 \
```
Comment (podlesrafal) [inline release/README.adoc:137]: mark somewhere that 4.3.0 needs to be changed to current release version

File: release/README.adoc [hunk 2]
```diff
@@ -124,3 +124,67 @@ flowchart LR
...
+  release-version
+----
+* curl HTTP
+[source, shell]
+----
+curl --location --request POST 'https://circleci.com/api/v2/project/gh/gravitee-io/gravitee-access-management/pipeline' \
+--header 'Content-Type: application/json' \
+--header "Circle-Token: $CIRCLECI_TOKEN" \
+--data-raw '{
+    "branch": "4.3.x",
```
Comment (podlesrafal) [inline release/README.adoc:148]: same here

File: release/README.adoc [hunk 3]
```diff
@@ -124,3 +124,67 @@ flowchart LR
...
+    "parameters": {
+        "gio_action": "release-alpha-version",
+        "dry_run": true,
+        "rc_requested": false,
+        "graviteeio_version": "4.3.3-alpha",
```
Comment (podlesrafal) [inline release/README.adoc:153]: and here. Same for `release-version`

File: release/steps/am-release-pipeline.mjs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { AbstractStepRelease } from './abstract-step-release.mjs';
+
+export class ReleaseVersion extends AbstractStepRelease {
```
Comment (podlesrafal) [inline release/steps/am-release-pipeline.mjs:3]: can you parameterize version here?

File: release/README.adoc (hunk 1 above)
Comment (mstasgravitee) [reply release/README.adoc:137]: it is only an example

File: release/README.adoc (hunk 2 above)
Comment (mstasgravitee) [reply release/README.adoc:148]: it is only an example

Comment (mstasgravitee) [reply release/README.adoc:148]: there is no branch for 4.4.x

File: release/README.adoc (hunk 3 above)
Comment (mstasgravitee) [reply release/README.adoc:153]: there is the next minor version for 4.3.x

**Review by podlesrafal — APPROVED** [review]

**Pull request merged**